dist/components/tabs/_tabs.dust

```diff
@@ -0,0 +1,20 @@
+{?tabs}
+    <div class="c-tabs" role="tab">
+        <ul id="c-tab-controls" class="c-tabs__controls" role="tablist">
```

Comment:
Not in love with the closeness between `c-tab-controls` and `c-tabs__controls`. Do we have a better `id` naming system?